gapic/templates/tests/unit/gapic/%name_%version/%sub/test_%service.py.j2

```diff
@@ -1053,10 +1066,13 @@ def test_transport_adc(transport_class):
 {% endfor %}
 {% for conf in configs %}
 {{ test_macros.transport_kind_test(**conf) }}
-
+{{ test_macros.run_transport_tests_for_config(**conf) }}
 {{ test_macros.empty_call_test(**conf) }}
 {% endfor %}
-
+{# TODO(https://github.com/googleapis/gapic-generator-python/issues/2121): Remove the below macro when async rest is GA. #}
```

Comment:
Is this TODO comment for the if statement or the actual macro?

Reply:
addressed by moving the TODO comment next to the macro.